cypress/pageobjects/virtualmachine.po.ts

```diff
@@ -210,4 +212,20 @@ export class VmsPage {
   usbTablet() {
     return new CheckboxPo('.checkbox-container', `:contains("Enable USB Tablet")`)
   }
+
+  efiEnabled() {
+    return new CheckboxPo('.checkbox-container', `:contains("Booting in EFI mode ")`)
+  }
+
+  deleteProgramlly(id:string) {
```

Comment:
What does `deleteProgramlly` mean? Is that a typo or something else?

Reply:
Like the login command, `deleteProgramlly` aims to delete resources directed by code instead of clicking on the UI.